cpu/sam0_common/periph/adc.c

```diff
@@ -176,18 +253,14 @@ int32_t adc_sample(adc_t line, adc_res_t res)
         DEBUG("adc: configuration failed\n");
         return -1;
     }
-#ifdef CPU_SAMD21
-    ADC->INPUTCTRL.reg = ADC_GAIN_FACTOR_DEFAULT |
+    ADC_DEV->INPUTCTRL.reg = ADC_GAIN_FACTOR_DEFAULT |
                          adc_channels[line].muxpos | ADC_NEG_INPUT;
-#else /* CPU_SAML21 */
-    ADC->INPUTCTRL.reg = adc_channels[line].muxpos | ADC_NEG_INPUT;
-#endif
     while (_adc_syncing()) {}
     /* Start the conversion */
```

Comment:
```suggestion

    /* Start the conversion */
```

separating the blocks with a newline helps readability